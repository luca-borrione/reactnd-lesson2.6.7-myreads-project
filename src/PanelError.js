import React from 'react';

const PanelError = () => {

	return (
		<div className="panel-error">
			Something went wrong. Please try again later.
		</div>
	);

};

export default PanelError;
