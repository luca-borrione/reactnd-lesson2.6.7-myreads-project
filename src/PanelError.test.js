import ReactDOM from 'react-dom';
import TestRenderer from 'react-test-renderer';
import PanelError from './PanelError';

describe('PanelError', () => {

	it('renders without crashing', () => {
		const div = document.createElement('div');
		ReactDOM.render(<PanelError />, div);
		ReactDOM.unmountComponentAtNode(div);
	});


	it('renders correctly', () => {
		const tree = TestRenderer
			.create(<PanelError />)
			.toJSON();

		expect(tree).toMatchSnapshot();
	});

});
