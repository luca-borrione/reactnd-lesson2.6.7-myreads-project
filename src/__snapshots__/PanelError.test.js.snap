// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PanelError renders correctly 1`] = `
<div
  className="panel-error"
>
  Something went wrong. Please try again later.
</div>
`;
